Automattic/sensei
# Fix progress bar and modules in Learning Mode not using global colors

Partial fix for https://github.com/Automattic/sensei-pro/issues/2115.

### Changes proposed in this Pull Request

This PR ensures that the progress bar and module names in Learning Mode work with global colors.

### Testing instructions

Please see https://github.com/Automattic/course/pull/176 as they should be tested together.
